### DISCUSSION TITLE
Update vim's official website

### DISCUSSION THREAD

**Discussion by Ali-Raza764** [vim/vim#15413] [Ideas]
I think the official website for vim is not very attractive, We can create a modern website to represent this amazing project using some meta framework like next.js what do you guys say ?

**Comment by tonymec** [comment]
I fear those who say, "Site XYZ is not very attractive, let's create an attractive site in its stead using ABCD".

The Vim website is what it is, and Vim old-timers immediately recognize it as what they have used for ages. If its look and feel changes, my first reaction will be: "Someone has hijacked the Vim site, let's get out of here quick !"

Regards,
Tony.

**Comment by chrisbra** [comment]
It's in the work: https://github.com/vim/website_next_generation (but I should check what the current status is)

**Reply by wilhelmtell** [reply]
Has that project fallen dormant?  What happened?  (Or what hasn't happened?)

**Reply by chrisbra** [reply]
It was a failed attempt unfortuantely

**Reply by wilhelmtell** [reply]
Oh, that's unfortunate. Do we know what happened? Has the team lost steam? Was it a politics or consensus issue? Time to put in vs value gained?
